src/authenticationservices.cs

```diff
@@ -1002,6 +1003,18 @@ enum ASAuthorizationSecurityKeyPublicKeyCredentialDescriptorTransport {
 		Bluetooth,
 	}
 
+	[Mac (12,0), iOS (15,0), MacCatalyst (15,0), NoWatch, NoTV]
+	[Native]
+	enum ASCOSEAlgorithmIdentifier : long {
+		Es256 = -7,
```

Comment:
but `ES256` since two-letters acronyms remains uppercase in .net
3+ letters acronyms become Pascal-cased